### DISCUSSION TITLE
Instructional Tasks - Craig Butikofer

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#215] [General]
I think that we are all really good at viewing the world from our own perspective. As simple as that sounds we all get really good at operating in our own sphere and thinking that others who operate are wrong. I think that what the video, Mihai's experience and the FreeCodeCamp lessons are getting at is that we need to consider who might be using our designs and how they might be using them. Sometimes making something a little more accessible is really simple and other times it can be more time intensive, but we can't get away from the idea that we need to consider how accessible our designs really are.

**Comment by kassyoveson** [comment]
I agree with you that how and what we design can enhance people's lives. Looking at the world from the perspective of your users will give you the best idea of how real people will react and use the design.

**Comment by gabebott** [comment]
This really drives home to me how important it is to consider empathy when designing. When you create something you always want to have a user centered mindset which will lead to empathetic design as well as accessible projects.
